### PR TITLE
Mintos currency

### PR DESCRIPTION
**Pull request by tobi-m**
Hi Chris,

thank you very much for sharing your neat tool and posting it on the forum of Portfolio Performance.

I added the functionality for different currencies on mintos. Please check if the changes are okay for the other p2p platforms.

Kind regards
Tobi

### CODE REVIEW

File: src/portfolio_performance_writer.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 import io
 import logging
 
-PP_FIELDNAMES = ['Datum', 'Wert', 'Typ', 'Notiz']
+PP_FIELDNAMES = ['Datum', 'Wert', 'Buchungswährung', 'Typ', 'Notiz']
```
Comment (ChrisRBe) [inline src/portfolio_performance_writer.py]: Hi Tobi,

Could you have a look at the unit tests?  The addition of this additional field breaks them.  :)

Comment (tobi-m) [reply src/portfolio_performance_writer.py]: Hi Chris,

thanks for the reply. I made the configuration of fee and currency optional and fixed the unit tests. There are checks failing, but for me it looks like project configuration issues which I can't fix.

Comment (ChrisRBe) [reply src/portfolio_performance_writer.py]: I believe I found the error for the failing unit test coverage upload :( 
https://docs.travis-ci.com/user/pull-requests/#Pull-Requests-and-Security-Restrictions
that is ... hmm ... too bad...

**Comment by tobi-m** [comment]
I don't get the code climate issue about the field "statement". Why should "statement" be renamed, when "config" doesn't?

Writing a get-method for every property in the Config class would fix the codacy issue, but it would blow up the code unnecessarily.

**Comment by ChrisRBe** [comment]
> I don't get the code climate issue about the field "statement". Why should "statement" be renamed, when "config" doesn't?
> 
> Writing a get-method for every property in the Config class would fix the codacy issue, but it would blow up the code unnecessarily.

I'm not entirely sure, but according to the documentation it has something to do that config in Config is not a member, and thus can't be accessed. statement on the other hand, since it's a member has this problem:
https://codeclimate.com/github/ChrisRBe/PP-P2P-Parser/pull/42#readup-5bc326ff969d05000100002e

I was thinking to try and rename self.statement to self._statement, maybe this will get rid of the finding as well?

**Comment by ChrisRBe** [comment]
@tobi-m looks good. I just hope that codacy will finish at some point :(

**Comment by tobi-m** [comment]
@ChrisRBe thank you for supporting my PR. The checks are successful, but a new hurdle has come up. Commits need to be signed. I'll have a look at it.

**Comment by ChrisRBe** [comment]
@tobi-m 👍 :)

**Pull request merged**